examples/tree_2d_dgsem/elixir_navierstokes_taylor_green_vortex.jl

```diff
@@ -16,8 +16,11 @@ equations_parabolic = CompressibleNavierStokesDiffusion2D(equations, mu=mu(),
 """
     initial_condition_taylor_green_vortex(x, t, equations::CompressibleEulerEquations2D)
 
-The classical inviscid Taylor-Green vortex, see for instance 
-https://en.wikipedia.org/wiki/Taylor%E2%80%93Green_vortex#Taylor%E2%80%93Green_vortex_solution
+The classical inviscid Taylor-Green vortex in 2D.
```

Comment:
You are still running a viscous test case, not an inviscid one, aren't you?

Reply:
See 3D.